src/app/data-planes/views/DataPlaneConfigView.vue

```diff
@@ -4,7 +4,7 @@
     name="data-plane-config-view"
     :params="{
       mesh: '',
-      dataPlane: ''
+      dataPlane: '',
```

Comment:
Same thing about linter

Reply:
Just to say I'm good with a lint rule to enforce trailing commas in JS

Reply:
This will be fixed in https://github.com/kumahq/kuma-gui/pull/1672. Apparently this rule needs to be turned on explicitly because it’s not part of the preset we’re using.